build/quine.js

```diff
@@ -0,0 +1,50 @@
+var through = require('through2');
+
+var source_variable = '__worker_src__'; // variable name used to refer to the Tangram source code
+var source_origin = '__worker_src_origin__'; // variable name used to refer to URL origin of source code (e.g., "http://localhost:8000/dist/tangram.debug.js")
+
+// prepend this code - wraps Tangram source in a function and later calls arguments.callee.toString to get source
+var prefix = '(function(){';
+    prefix += 'var target = (typeof module !== "undefined" && module.exports) || (typeof window !== "undefined");';
+    prefix += 'if (target) {';
+    prefix += 'var ' + source_variable + ' = arguments.callee.toString();';
+    prefix += 'var ' + source_origin + ' = document.currentScript.src;';
```

Comment:
`document.currentScript` is not available in IE (http://caniuse.com/#feat=document-currentscript), so we will need to guard against it being null here.